### PR TITLE
Add Vietnamese Translation

### PR DESCRIPTION
**Pull request by nhatientri**
first version of vietnamese

### CODE REVIEW

**Comment by ironwallaby** [comment]
Thanks for this! Can you also please contribute a `/test_cases/vi.json` so that we can make sure the translation works as expected, too?

**Comment by nhatientri** [comment]
oh sorry, i've just got some error, i'll contribute files weekend

**Comment by phungdoanh** [comment]
Could you please add me into translate vietnamese

File: test_cases/vi.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+  "Quang mây":
+    ["title", "clear"],
+
+  "Có thể có mưa nhẹ":
+    ["title", "possible-very-light-precipitation"],
+
+  "Mưa nhẹ":
+    ["title", "very-light-precipitation"],
+
+  "có thể có mưa nhẹ":
```
Comment (vtabk2) [inline test_cases/vi.json]: Có thể có mưa nhẹ

File: test_cases/vi.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  "có tuyết rơi":
+    ["title", "medium-sleet"],
+
+  "tuyết rơi":
+    ["title", "heavy-sleet"],
+
+  "Có thể có tuyết rơi nhẹ":
+    ["title", "possible-very-light-snow"],
+
+  "gió nhẹ":
```
Comment (vtabk2) [inline test_cases/vi.json]: Tuyết rơi nhẹ

File: test_cases/vi.json [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    ["title", ["and", "high-humidity", "light-clouds"]],
+
+
+  "thỉnh thoảng quang mây":
+    ["sentence", ["for-hour", "clear"]],
+
+  "có dông sau 35 phút":
+    ["sentence", ["starting-in", "very-light-snow", ["minutes", 35]]],
+
+  "hết dông sau 15 phút":
```
Comment (vtabk2) [inline test_cases/vi.json]: Hết mưa sau 15 phút

File: test_cases/vi.json [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    ["sentence", ["stopping-in", "light-rain", ["minutes", 15]]],
+
+  "Heavy sleet starting in 20 min., stopping 30 min. later.":
```
Comment (vtabk2) [inline test_cases/vi.json]: ??

Comment (vtabk2) [reply test_cases/vi.json]: Mưa to bắt đầu trong 20 phút và dừng lại sau khi mưa 30 phút

File: test_cases/vi.json [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+  "tuyết nhẹ (2 cm.) sau buổi sáng.":
+    ["sentence", ["during",
+      ["parenthetical", "light-snow", ["centimeters", 2]],
+      "later-today-morning"]],
+
+  "tuyết rơi (8\u201312 in.) cả ngày.":
+    ["sentence", ["for-day",
+      ["parenthetical", "heavy-snow", ["inches", ["range", 8, 12]]]]],
+
+  "tuyết rơi (under 1 cm.) vào chiều nay.":
```
Comment (vtabk2) [inline test_cases/vi.json]: Tuyết rơi (dưới 1 cm) vào chiều nay

**Comment by vtabk2** [comment]
missing Humid and Mostly Cloudy

**Comment by ironwallaby** [comment]
It's been over a year, so I'm going to close this. Please resubmit with the requested fixes if you remain interested!

**Pull request closed without merging**